src/pspm_cfg/pspm_cfg_help_format.m

```diff
@@ -0,0 +1,39 @@
+function helptext = pspm_cfg_help_format(funcname, argname)
+% pspm_cfg_help_format collects help text elements generated by
+% pspm_help_init and combines them into a text block for the matlabbatch
+% GUI
+% Format: helptext = pspm_cfg_help_format(funcname, [argname])
+%                funcname: PsPM function name (char)
+%                argname: function argument (potentially a chain of nested
+%                struct fields, e.g. 'options.overwrite')
+global settings
+
+if nargin < 2
+    helptext = {
+        '------------------------------------------------------------------------------------------------', ...
+        sprintf(...
+        ['This GUI item calls the function %s. You can also call this function ', ...
+         'directly. Type ''help %s'' in the command window for more information.'], funcname, funcname), ...
+        '------------------------------------------------------------------------------------------------'};
+    
+    if isfield(settings.help, funcname)
+        if isfield(settings.help.(funcname), 'Description')
+            A = settings.help.(funcname).Description;
+            A = strrep(A, newline, [newline, newline]);
+            A = splitlines(A);
+            helptext = [helptext(:); ''; A];
+        end
+        if isfield(settings.help.(funcname), 'References')
+            helptext = [ helptext(:); ''; ...
+            '------------------------------------------------------------------------------------------------';...
+            'References:'; ...
+            settings.help.(funcname).References(:)];
+        end
+    end
+else
+    % this syntax allows chaining several nested structs into one argname
+    evalc(sprintf('helptext = settings.help.%s.Arguments.%s;', funcname, argname));
+    [startindx, endindx] = regexp(helptext, '\[\s*([^\[\]]*)\s*\]'); % thanks ChatGPT for finding the regexp
+    helptext(startindx:endindx) = [];
```

Comment:
`startindx` and `endindx` could be 1xn (n>1) array. I would suggest to use `transpose` in this occasion. Please find screenshot.

<img width="573" alt="image" src="https://github.com/user-attachments/assets/0a159d29-b05c-4412-97f5-57ddece05a9b">

Reply:
fixed.